### ISSUE TITLE
Check failed: stream->parent()->GetConvolveAlgorithms(&algorithms)

### ISSUE THREAD

**Issue by ElNandes** [tensorflow/tensorflow#12478]
INFO:tensorflow:Creating bottleneck at /tmp/bottleneck\dandelion\10043234166_e6dd915111_n.jpg_inception_v3.txt
2017-08-22 12:36:45.710892: W c:\tf_jenkins\home\workspace\release-win\m\windows-gpu\py\35\tensorflow\core\framework\op_def_util.cc:332] Op BatchNormWithGlobalNormalization is deprecated. It will cease to work in GraphDef version 9. Use tf.nn.batch_normalization().
2017-08-22 12:36:46.351813: E c:\tf_jenkins\home\workspace\release-win\m\windows-gpu\py\35\tensorflow\stream_executor\cuda\cuda_dnn.cc:359] could not create cudnn handle: CUDNN_STATUS_NOT_INITIALIZED
2017-08-22 12:36:46.351943: E c:\tf_jenkins\home\workspace\release-win\m\windows-gpu\py\35\tensorflow\stream_executor\cuda\cuda_dnn.cc:366] error retrieving driver version: Unimplemented: kernel reported driver version not implemented on Windows
2017-08-22 12:36:46.354865: E c:\tf_jenkins\home\workspace\release-win\m\windows-gpu\py\35\tensorflow\stream_executor\cuda\cuda_dnn.cc:326] could not destroy cudnn handle: CUDNN_STATUS_BAD_PARAM
2017-08-22 12:36:46.356828: F c:\tf_jenkins\home\workspace\release-win\m\windows-gpu\py\35\tensorflow\core\kernels\conv_ops.cc:671] Check failed: stream->parent()->GetConvolveAlgorithms(&algorithms)

**Comment by ali01** [comment]
Please provide details about what platform you are using  (operating system, architecture). Also include your TensorFlow version. Also, did you compile from source or install a binary?  Make sure you also include the exact command if possible to produce  the output included in your test case. If you are unclear what to include  see the issue template displayed in  [the Github new issue template](https://github.com/tensorflow/tensorflow/issues/new).

 We ask for this in the issue submission template, because    it is really difficult to help without that information. Thanks!

**Comment by ElNandes** [comment]
I had tried to run the label_image.py in terminal but the issue was solved after I updated to Tensorflow 1.3

**Comment by michaelisard** [comment]
Good to hear the issue is resolved. I'll close for now.

**Comment by XiaoYigwr** [comment]
When I use the keras==1.2.0, I have got the same problem. Fortunately,  the issue was solved after I upgrade the tensorflow from 1.2.0 to 1.3.0 .
